assets/js/main.js

```diff
@@ -12,7 +12,7 @@ var Altis = window.Altis || {}; // eslint-disable-line no-var
 Altis.Consent = {
 	giveConsentButton: document.querySelector( '.give-consent' ),
 	revokeConsentButton: document.querySelector( '.revoke-consent' ),
-	cookiePreferences: document.querySelector( '.cookie-preferences' ).classList,
+	cookiePreferences: document.querySelector( '.cookie-preferences' ) ? document.querySelector( '.cookie-preferences' ).classList : false,
```

Comment:
Haven't validated whether the babel config in use here is new enough for this, but if your build supports [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) that would be a bit tidier:
```suggestion
	cookiePreferences: document.querySelector( '.cookie-preferences' )?.classList || false,
```

Reply:
I was wondering what the easier way to do this was. I will give this a shot 👍

Reply:
It didn't support it (the babel config is super basic), but I added it and it works 🎉
thanks again!